### DISCUSSION TITLE
Hosting MonkeyType on Raspberry Pi

### DISCUSSION THREAD

**Discussion by dustinormond** [monkeytypegame/monkeytype#3577] [Questions] [answered]
I really like monkeytype and all its options. We develop a mini-computer (i.e. BlueBox) that contains a bunch of educational materials where users in rural areas without access to the Internet can access the content on the BlueBox. We would love to add monkeytype to the list. I have a few questions though:

1. Is this something we can and are allowed to do?
2. What would I need to install to get the web server to recognize monkeytype and render its content?

For more information about the BlueBox project, visit bluebox.creighton.edu or bluebox.creighton.edu/demo to see a demo.

**Comment by Miodec** [comment, marked as answer]
You can follow the guide here: https://github.com/monkeytypegame/monkeytype/blob/master/CONTRIBUTING_ADVANCED.md

You can probably skip the database stuff and just run the frontend

**Reply by Miodec** [reply]
Now that i think about it, it would probably be beneficial for you to also skip the Firebase step. You could probably just build the frontend, then copy the build `public` folder to a simple Express app that serves static files

Feel free to ask more questions if you need to
